README.md

```diff
@@ -1,3 +1,30 @@
 # niftimath-rs
 
-NiftiMath rust project.
\ No newline at end of file
+NiftiMath rust project.
+
+This project provide a simple way to make mathematical operation on nifti images (the 
+same way mrcalc from [mrtrix3](https://github.com/MRtrix3/mrtrix3)).
+
+The formulas are express in [reverse polish notation](https://en.wikipedia.org/wiki/Reverse_Polish_notation).
+
+## Exemple
+```
+niftimath 1.0 t1.nii.gz mul 3 6.1 add mul output.nii.gz
+```
+This correspond to the following operation:
+```
+output.nii.gz = (1.0 * t1.nii.gz) * (3 + 6.1)
```

Comment:
Good idea! Maybe try to modify this example (or add new ones) when you implement new operators?

Reply:
For sure